test/e2e/storage/testsuites/multivolume.go

```diff
@@ -591,6 +692,49 @@ func TestConcurrentAccessToSingleVolume(f *framework.Framework, cs clientset.Int
 	}
 }
 
+// TestConcurrentAccessToRelatedVolumes tests access to multiple volumes from multiple pods.
+// Each provided PVC is used by a single pod. The test ensures that volumes created from
+// another volume (=clone) or volume snapshot can be used together with the original volume.
+func TestConcurrentAccessToRelatedVolumes(f *framework.Framework, cs clientset.Interface, ns string,
+	node e2epod.NodeSelection, pvcs []*v1.PersistentVolumeClaim, requiresSameNode bool,
+	readOnly bool) {
+
+	var pods []*v1.Pod
+
+	// Create each pod with pvc
+	for i := range pvcs {
+		index := i + 1
+		ginkgo.By(fmt.Sprintf("Creating pod%d with a volume on %+v", index, node))
+		podConfig := e2epod.Config{
+			NS:            ns,
+			PVCs:          []*v1.PersistentVolumeClaim{pvcs[i]},
+			SeLinuxLabel:  e2epod.GetLinuxLabel(),
+			NodeSelection: node,
+			PVCsReadOnly:  readOnly,
+			ImageID:       e2epod.GetTestImageID(imageutils.JessieDnsutils),
+		}
+		pod, err := e2epod.CreateSecPodWithNodeSelection(cs, &podConfig, f.Timeouts.PodStart)
+		defer func() {
+			framework.ExpectNoError(e2epod.DeletePodWithWait(cs, pod))
+		}()
+		framework.ExpectNoError(err)
+		pods = append(pods, pod)
+		actualNodeName := pod.Spec.NodeName
+
+		// Set affinity depending on requiresSameNode
+		if requiresSameNode {
```

Comment:
I see that `requiresSameNode` is false in the tests, what happens when anti-affinity is set after the pod is created with `CreateSecPodWithNodeSelection`? is it rescheduled again?

Reply:
Removed `requiresSameNode` and `readOnly` param of `TestConcurrentAccessToRelatedVolumes` in https://github.com/kubernetes/kubernetes/pull/102775